localnet/run.ts

```diff
@@ -0,0 +1,351 @@
+import { LOCALHOST } from '@metaplex-foundation/amman';
+import {
+  Amman,
+  ammanMockStorage,
+  AmmanMockStorageDriver,
+} from '@metaplex-foundation/amman-client';
+import { Connection, Keypair, PublicKey } from '@solana/web3.js';
+import path from 'path';
+import * as fs from 'fs';
+import {
+  Metaplex,
+  keypairIdentity,
+  toMetaplexFile,
+  Nft,
+} from '@metaplex-foundation/js';
+
+import { CreateFanout } from '../src/graphql';
+import * as graphqlTypes from '../src/graphql';
+import { makeSchema } from 'nexus';
+import { resolveObjMapThunk } from 'graphql';
+
+import { asNexusMethod } from 'nexus';
+import { makeServer } from '../src/makeServer';
+import { GraphQLClient, gql, RequestDocument } from 'graphql-request';
+import base58 from 'bs58';
+import { defaultYogaLogger } from '@graphql-yoga/common';
+import { Fanout, FanoutClient } from '@glasseaters/hydra-sdk';
+import { Wallet } from '@project-serum/anchor';
+
+const ensureBalance = async (
+  amman: Amman,
+  connection: Connection,
+  pubkey: PublicKey,
+  minBalance: number,
+  maxBalance: number = 2 * minBalance,
+) => {
+  const currentBalance = await connection.getBalance(pubkey, 'confirmed');
+  if (currentBalance < minBalance) {
+    const sig = await amman.airdrop(connection, pubkey, maxBalance);
+    // NOTE(will): need to finzalize otherwise tx's done immediately after will fail
```

Comment:
delete?

```suggestion
```

Reply:
this is a comment explaining that we need to wait for the tx to finalize here. i'd prefer to just leave it to remember why thats there.